# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/vlc/libsmbclient.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur vlc-0.8.2/modules/access/smb.c src.vlc.1128277340.4374.1626276738/vlc-0.8.2/modules/access/smb.c
--- vlc-0.8.2/modules/access/smb.c	2005-06-25 15:43:11.000000000 +0200
+++ src.vlc.1128277340.4374.1626276738/vlc-0.8.2/modules/access/smb.c	2005-10-02 20:59:28.000000000 +0200
@@ -301,7 +301,7 @@
     access_sys_t *p_sys = p_access->p_sys;
 
 #ifdef USE_CTX
-    p_sys->p_smb->close( p_sys->p_smb, p_sys->p_file );
+    p_sys->p_smb->close_fn( p_sys->p_smb, p_sys->p_file );
     smbc_free_context( p_sys->p_smb, 1 );
 #else
     smbc_close( p_sys->i_smb );
